lib/catFile.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var through = require('through2');
+var gutil = require('gulp-util');
+var exec = require('child_process').exec;
+var escape = require('any-shell-escape');
+var stripBom = require('strip-bom-stream');
+var BufferStreams = require('bufferstreams');
+
+module.exports = function (opt) {
+  if (!opt) opt = {};
+  if (null == opt.stripBOM) opt.stripBOM = true;
```

Comment:
Please do `variable | comparator` as in `if (opt === null)`
Also use triple equals `===`, `!==` as the other files have.